docs/getting-started.md

```diff
@@ -1,88 +1,94 @@
-> [!IMPORTANT]
-> The docs is Work In Progress `WIP`, and the content is not final.
+# Getting Started with QSE
 
-# Markdown Extension Examples
+Quick Start Express (QSE) is a simple yet powerful CLI tool designed to quickly generate Express servers from multiple templates. It simplifies server setup with customizable configurations including server name, Docker Compose file generation, and more.
 
-This page demonstrates some of the built-in markdown extensions provided by VitePress.
+## Installation
 
-## Syntax Highlighting
+Install the `quick_start_express` package globally via npm. Run the following command in the terminal:
 
-VitePress provides Syntax Highlighting powered by [Shiki](https://github.com/shikijs/shiki), with additional features like line-highlighting:
+```shell
+npm i -g quick_start_express
+```
 
-**Input**
+> **:exclamation: Important:** A global install (with `-g`) is required for `qse` to function.
```

Comment:
You can make use of https://vitepress.dev/guide/markdown#github-flavored-alerts for better UI for this.

Here and places below too.